torch/cuda/nvtx.py

```diff
@@ -0,0 +1,48 @@
+import os
+import ctypes
+import warnings
+import torch.cuda
+
+lib = None
+
+__all__ = ['range_push', 'range_pop', 'mark']
+
+def _libnvToolsExt():
+    global lib
+    if lib is None:
+        # nvToolsExt.so is lazily loaded, so if you want to use these tools
+        # the library needs to be in your LD_LIBRARY_PATH
+        lib = ctypes.cdll.LoadLibrary("libnvToolsExt.so")
```

Comment:
Do we distribute all of them? I thought these are just the ones we use? If not, we should remove all unused libs, as it will make the binaries smaller.